core/src/main/java/hudson/model/ComputerSet.java

```diff
@@ -410,6 +411,21 @@ public void doRun() {
         }, 10, TimeUnit.SECONDS);
     }
 
+    /**
+     * @return The list of strings of computer names (excluding master)
+     * @since TODO
+     */
+    @Nonnull
+    public static List<String> getComputerNames() {
+        final ArrayList<String> names = new ArrayList<String>();
+        for (Computer c : Jenkins.getActiveInstance().getComputers()) {
```

Comment:
Yes, we did. But we do not really expect null instance to happen on master for extension points